scripts/claude-github-actions-check.js

```
@@ -0,0 +1,252 @@
+// Command line interface
+const command = process.argv[2];
+const detailed = process.argv.includes('--detailed');
+
+console.log('🤖 Claude GitHub Actions Check');
+console.log('='.repeat(50));
+
+switch (command) {
+  case 'status':
+    const isHealthy = checkBuildMonitoring();
+    checkMissingRuns();
+    console.log(`\n🏥 Overall Status: ${isHealthy ? '✅ HEALTHY' : '⚠️  NEEDS ATTENTION'}`);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)